debug_toolbar/decorators.py

```diff
@@ -10,7 +10,11 @@ def inner(request, *args, **kwargs):
 
         show_toolbar = get_show_toolbar()
         if not show_toolbar(request):
-            raise Http404
+            raise Http404(
+                'You do not have the permission to access debug-toolbar'
+                ' urls. Please check your INTERNAL_IPS and'
+                ' SHOW_TOOLBAR_CALLBACK configurations'
```

Comment:
The idea behind raising a 404 instead of a 403 is that unprivileged users shouldn't even be aware of Django debug toolbar's existence. From the user's perspective, the resource "doesn't exist" instead of "the resource exists, but you aren't authorized to access it". This message looks to be leaking this information now. I think it shouldn't.